Makefile

```diff
@@ -1,8 +1,8 @@
-## This Makefile is meant to be run on Unix hosts - as such, it only supports the few
-## operations that don't require on a Windows host, mostly code generation and linting.
+os=windows
+CMDS=server
+all: build test-go
 
-.DEFAULT_GOAL := all
-SHELL := /bin/bash
+include release-tools/build.make
 
 ifeq ($(GOPATH),)
```

Comment:
Tip: instead of requiring that GOPATH is set, initialize it:
```
GOPATH ?= $(shell go env GOPATH)
```

Then the Makefile will also works for those developers who don't have `GOPATH` in their environment.

Reply:
fixed it. thanks!

Reply:
fixed it. thanks!